### ISSUE TITLE
Cannot pull tanka docker image because of unknown manifest.

### ISSUE THREAD

**Issue by Estrax** [grafana/tanka#89]
**What happened**:

Cannot pull tanka docker image because of unknown manifest.

**What you expected to happen**:

Tanka docker image pulling down properly.

**How to reproduce it (as minimally and precisely as possible)**:

1. Go to https://github.com/grafana/tanka/releases
2. Look at docker container section
3. run `docker pull grafana/tanka:v0.5.0`

Result:
```
> docker pull grafana/tanka:v0.5.0`
Error response from daemon: manifest for grafana/tanka:v0.5.0 not found: manifest unknown: manifest unknown
```

**Anything else we need to know?**:

**Environment**:
- User OS: Ubuntu 18.04, Ubuntu 19.04, Debian 9, macOS 10.14.6 (18G103), macOS 10.15 GM (19A582a)

**Comment by sh0rez** [comment]
Hi! Thanks for reporting, the CI is misconfigured and strips the `v` from the tags name. Changing it, use `grafana/tanka:0.5.0` for the time being

**Comment by sh0rez** [comment]
Solved in #90

**Comment by dohnto** [comment]
@sh0rez In the release page, you still use prefix `v`, even for version 0.6.0.
